INSTALL.rst

```diff
@@ -15,16 +15,6 @@ this:
 
    $ pip install invenio-rest[cors]
 
-Configuration
--------------
-
-===================== =================================================
-`REST_ENABLE_CORS`    Set to ``True`` to enable Cross-Origin Resource
-                      Sharing. Defaults to ``False``.
-`REST_CORS_RESOURCES` Dictionary for configuring CORS for endpoints.
-                      See Flask-CORS for further details.
```

Comment:
added in the config file here: https://github.com/inveniosoftware/invenio-rest/pull/63/files#diff-944546f76b8cc3d9d71b4cb5316b0ad6R30